dashboard/app/dsl/content_dsl.rb

```diff
@@ -38,17 +38,14 @@ def method_missing(key, *args)
     @hash[:options][key.to_sym] = args.first
   end
 
-  def i18n_strings
-    strings = {}
-    %i(
-      title
+  # @override
+  def self.i18n_fields
+    super + %w(
       content1
       content2
       content3
-      pre_title
```

Comment:
Quick double check that `pre_title` should be removed.

Reply:
yep! See https://github.com/code-dot-org/code-dot-org/blob/82c978b50bfe24f16f4eb2d00df2db08752b7383/dashboard/app/dsl/content_dsl.rb#L30-L34